source/SIL.AppBuilder.Portal.Frontend/src/ui/styles/semantic-ui-overrides.scss

```diff
@@ -180,6 +180,14 @@ Dropdown Menu
   }
 }
 
+.ui.dropdown.is-large {
```

Comment:
from the name of the class I expected width:100% or something like that... but it's more related to font size large